packages/turbo/ios/RNVisitableViewController.swift

```diff
@@ -24,36 +24,82 @@ public protocol RNVisitableViewControllerDelegate {
   
 }
 
-class RNVisitableViewController: VisitableViewController {
-  
-  public var delegate: RNVisitableViewControllerDelegate?
-  
+class RNVisitableViewController: UIViewController, Visitable {
+  public var visitableViewControllerDelegate: RNVisitableViewControllerDelegate?
+
+  open weak var visitableDelegate: VisitableDelegate?
+  open var visitableURL: URL!
+
+  private var reactViewController: UIViewController? = nil
+
+  public convenience init(url: URL) {
+    self.init()
+    self.visitableURL = url
+  }
+    
+  public convenience init(reactViewController: UIViewController?, visitableViewControllerDelegate: RNVisitableViewControllerDelegate?) {
+    self.init()
+    self.reactViewController = reactViewController
+    self.visitableViewControllerDelegate = visitableViewControllerDelegate
+  }
+
+  // MARK: View Lifecycle
+
+  override func viewDidLoad() {
+    super.viewDidLoad()
+    view.backgroundColor = UIColor.white
```

Comment:
Is this needed?

Reply:
If we remove that we won't attach the webview :D

Reply:
I mean the background color